components/camel-aws/camel-aws2-s3/src/main/java/org/apache/camel/component/aws2/s3/AWS2S3Producer.java

```diff
@@ -214,7 +214,10 @@ public void processMultiPart(final Exchange exchange) throws Exception {
 
                 LOG.trace("Uploading part [{}] for {}", part, keyName);
                 try (InputStream fileInputStream = new FileInputStream(filePayload)) {
-                    fileInputStream.skip(filePosition);
+                    long skipped = fileInputStream.skip(filePosition);
+                    if (skipped == 0) {
+                        LOG.warn("0 bytes skipped");
```

Comment:
I'd probably use a more descriptive log message here.

Reply:
please elaborate on your requirement

Reply:
I mean something along the lines of: `LOG.warn("While trying to upload the file {} file, 0 bytes were skipped", keyName);`. 

You have to consider that the previous log message of at line 215 is at a trace level. As such, the user would only have this on the logs: `0 bytes skipped`  which is not sufficient for a user to understand what happened with what resource under what condition.

Reply:
thanks, got it!